src/index.js

```diff
@@ -1,7 +1,48 @@
+const client = require('prom-client');
 const config = require('config');
 const fetchStatuCakeData = require('./statuscake');
 const server = require('./server');
+const Storage = require('./statuscakedata');
 
-this.interval = setInterval(fetchStatuCakeData, config.statuscake.interval);
+const regex = /\s?([-])\s?/;
+
+const getTagsFromTitle = Title => Title.split(regex);
+
+const g = new client.Gauge({
+  name: 'metric_loadtime',
+  help: 'metric_help',
+  labelNames: ['customer', 'service', 'device', 'version'],
+});
+const h = new client.Gauge({
+  name: 'metric_filesize',
+  help: 'metric_help',
+  labelNames: ['customer', 'service', 'device', 'version'],
+});
+const i = new client.Gauge({
+  name: 'metric_request',
+  help: 'metric_help',
+  labelNames: ['customer', 'service', 'device', 'version'],
+});
+
+setInterval(() => {
+  const date = (100, Date.now());
+  const data = Storage.getData();
+  if (data.length > 0) {
+    data.forEach((myItem) => {
+      const parsedTitle = getTagsFromTitle(myItem.Title);
+      g.set({
```

Comment:
The 3 gauge set look exactly the same. We usually avoid code duplication in order to improve readability of code.

It should be great to extract the duplicated code in a function 💡

Reply:
Exact, it's done. Thanks 👍